packages/blitz-auth/src/client/index.tsx

```diff
@@ -289,9 +298,11 @@ function withBlitzAuthPlugin<TProps = any>(Page: ComponentType<TProps> | BlitzPa
           const url = new URL(redirectTo, window.location.href)
           url.searchParams.append("next", window.location.pathname)
           debug("[BlitzAuthInnerRoot] redirecting to", url.toString())
```

Comment:
move this log into the if statement (and might as well add the other url code in there also)